### ISSUE TITLE
Issue with mailbox:letterbox (see image for further intel)

### ISSUE THREAD

**Issue by Lanhild** [obl3pplifp/BlS-issue-tracker#371]
![image](https://user-images.githubusercontent.com/73610779/149840348-cd6719f0-6d39-47b3-98a2-29b836d05a71.png)

Honestly don't know why this happens

**Comment by oversword** [comment]
@Lanhild This is someone else's mailbox right?

**Comment by Lanhild** [comment]
Yeah, i think it's just because i don't have the right privilege, just wanted to make sure.

**Comment by oversword** [comment]
@Lanhild You have the right privilege, it wouldn't show you the message if you didn't. I added in that message because it's not possible to access the inventory while the player is not logged in. It's a problem, but it's known about.. hence the message telling you it's an issue :P

**Comment by Lanhild** [comment]
Oh l see, it's more clear now. Shall i close the issue then?

**Comment by oversword** [comment]
@Lanhild you can leave it open if you like, just don't expect it to be solved any time soon
